sdks/python/setup.py

```diff
@@ -112,7 +112,7 @@ def get_version():
     author=PACKAGE_AUTHOR,
     author_email=PACKAGE_EMAIL,
     packages=setuptools.find_packages(),
-    package_data={'apache_beam': ['**/*.pyx', '**/*.pxd']},
+    package_data={'apache_beam': ['**/*.pyx', '**/*.pxd', 'tests/data/*']},
```

Comment:
Can you document how this key was generated somewhere. May be `tests/data/README.md`